ovirt_imageio/_internal/auth.py

```diff
@@ -433,23 +440,24 @@ def authorize(self, ticket_id, op):
         """
         Authorizing a ticket operation
         """
-        log.debug("AUTH op=%s ticket=%s", op, ticket_id)
         try:
             ticket = self._tickets[ticket_id]
         except KeyError:
             raise errors.AuthorizationError(
                 "No such ticket {}".format(ticket_id))
 
+        log.debug("AUTH op=%s transfer=%s", op, ticket.transfer_id)
+
         if ticket.canceled:
             raise errors.AuthorizationError(
-                "Ticket {} was canceled".format(ticket_id))
+                "Transfer={} was canceled".format(ticket.transfer_id))
```

Comment:
Maybe instead of:
    Transfer={}
better format is:
    Transfer '{}'

Reply:
Yes, this was a mistake, the intent was to keep the format, just replace Ticket with Transfer.